Numerics/CustomModAdd/CustomModAdd.qs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Numerics.Samples
+{
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Arithmetic;
+    open Microsoft.Quantum.Diagnostics;
+
```

Comment:
If reasonable, I'd suggest a sequence of comments here (`//` rather than documentation comments) explaining the purpose of this sample and introducing a bit, so that the reader understands the context and knows what to expect going forward.